src/OrleansRuntime/OrleansRuntime.csproj

```diff
@@ -119,6 +119,7 @@
     </Compile>
     <Compile Include="ReminderService\MockReminderTable.cs" />
     <Compile Include="Scheduler\ITaskScheduler.cs" />
+    <Compile Include="Startup\ConfigureStartupBuilder.cs" />
```

Comment:
`ConfigureServiceBuilder ` is statefull class, it has a member field that is being reused (`MethodInfo`), while `ConfigureStartupBuilder` is just a bunch of static methods. Can those unrelated static method be put inside `ConfigureServiceBuilder `? Maybe. But that is not what this PR is about. This PR is about keeping the other code, prio to DI, clean. A nicer refactoring of the DI code is left for future. I think @attilah may be planning to do more work on that.